css/style.css

```diff
@@ -1,45 +1,99 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@200;300&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500&display=swap');
+
 * {
   font-family: "Poppins", sans-serif;
   margin: 0;
   padding: 0;
 }
 
-body {
-  color: #f5f5ff;
-  background: #191765;
+.mobile-menu-container {
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: 0;
+  background-color: rgba(25, 23, 101, 1);
+  background-image: url("../img/shapes\ mobile\ menu.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 100vh;
+  width: 100%;
+  padding: 30px;
+  visibility: hidden;
 }
 
-a {
-  text-decoration: none;
+.mobile-menu-ul {
+  margin-top: 80px;
 }
 
-li {
-  list-style: none;
+.mobile-menu-ul a {
+  color: #fff;
+  margin: 15px 0;
+  margin-left: 10%;
+  display: block;
 }
 
-.endnavbar {
-  padding: 20px;
-  justify-content: end;
-  color: #f9f9f9;
+.mobile-menu-close {
+  display: block;
+  text-align: right;
+  margin-right: 10%;
+  margin-top: 10%;
 }
 
-.container {
-  max-width: 900px;
-  margin: auto;
-  width: 100%;
+.mobile-menu-li {
+  margin-bottom: 20px;
+}
+
+.mobile-menu-a {
+  position: absolute;
+  top: 40px;
+  right: 40px;
+  width: 30px;
```

Comment:
- [ ] Kindly make sure your mobile menu matches the Figma design by following the font size, padding, and adding an arrow icon like in Figma.

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/201992318-da0835d7-f229-4369-8efc-66dd128fc970.png)  | ![image](https://user-images.githubusercontent.com/53402600/201992420-47b1e474-15cf-4812-a9bd-0e0bfc40641a.png)|